TailLight/driver.cpp

```diff
@@ -11,13 +11,16 @@ NTSTATUS DriverEntry(
 
     WDF_DRIVER_CONFIG params = {};
     WDF_DRIVER_CONFIG_INIT(/*out*/&params, EvtDriverDeviceAdd);
-    params.DriverPoolTag = POOL_TAG;
```

Comment:
Why did you remove this line?

Reply:
There was a previous comment that this should be only added in a separate PR. I built a task list and was using e-mail to respond to these issues. I ensured that this was removed as I had a task for it. Hopefully continued practice with rebasing and merging will reduce these kinds of issues. Please verify that it's back in.

Reply:
Thanks.